emperor/support_files/js/controller.js

```diff
@@ -435,7 +435,13 @@ define([
     });
 
     this.$plotBanner.css({'color': color, 'border-color': color});
-    this.$plotBanner.html(visible + '/' + total + ' visible');
+
```

Comment:
Should there be a test for this?

Reply:
Unfortunately not now, as we don't have a good way to test the `EmperorController` object.